lemur/notifications/messaging.py

```diff
@@ -96,11 +97,20 @@ def send_notification(event_type, data, targets, notification):
     :param notification:
     :return:
     """
+    function = f"{__name__}.{sys._getframe().f_code.co_name}"
+    log_data = {
+        "function": function,
+        "message": "Sending expiration notification for to targets {}".format(targets),
```

Comment:
This is optional, instead of `format()` I have found `f"` notation to be more concise and readable for string formatting. Same as the one from line 100 ([here](https://github.com/Netflix/lemur/blob/master/lemur/certificates/schemas.py#L197) is another example). It was suggested to me by Hossein in my PR.

Reply:
yeah, this kinda makes for a good readability, and we have been adopting this approach in the last 2+ years, despite a lot of legacy code still being in the old format.

Reply:
Yeah, it's a much better style. I updated all the strings in this file to use that.